src/main/java/ch/njol/skript/expressions/ExprCpuUsage.java

```diff
@@ -0,0 +1,112 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import me.lucko.spark.api.Spark;
+import me.lucko.spark.api.SparkProvider;
+import me.lucko.spark.api.statistic.StatisticWindow;
+import me.lucko.spark.api.statistic.types.DoubleStatistic;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+
+@Name("CPU (Central Processing Unit) Usage")
+@Description("Returns the 3 most recent CPU Usage readings, like the information from Spark's /tps command." +
+			"This expression is only supported with servers that have Spark on their server.")
```

Comment:
This will output:
Returns the 3 most recent CPU Usage readings, like the information from Spark's /tps command.This expression is only supported with servers that have Spark on their server.